src/components/protocol/src/rpc_type.cc

```diff
@@ -62,6 +62,7 @@ RpcType RpcTypeFromByte(uint8_t byte) {
   return supported_type ? type : kRpcTypeReserved;
 }
 
+// cppcheck-suppress unusedFunction
 uint8_t RpcTypeToByte(RpcType type) {
```

Comment:
@Ypostolov It seems like the function is unused. Please, check it and remove if it is true.